### PR TITLE
get_volunteer_by_id not found error solved while editing profile

### LINKED ISSUE #561: Error in editing profile of a volunteer
## Description
get_volunteer_by_id needs to be imported from services file. 

## Mocks
screenshot from 2018-01-26 13-52-03
image

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
get_volunteer_by_id and has_resume_file is imported in views from services.

Fixes #561 

# Type of Change:
- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
screenshot from 2018-01-26 14-11-44

# Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak @tonythomas01 Please review.

File: vms/volunteer/views.py [hunk 1]
```diff
@@ -1,242 +1,242 @@
...
+from shift.services import get_volunteer_report, calculate_total_report_hours
+from volunteer.forms import ReportForm, SearchVolunteerForm, VolunteerForm
+from volunteer.models import Volunteer
+from volunteer.services import get_volunteer_by_id, delete_volunteer_resume,search_volunteers, get_volunteer_resume_file_url, has_resume_file
+from volunteer.validation import validate_file
+from volunteer.utils import vol_id_check
+from vms.utils import check_correct_volunteer
+
+
+@login_required
```
Comment (tonythomas01) [inline vms/volunteer/views.py]: Why do you have all this unrelated changes here ? 

This should just be a 2 line PR and nothing else.

Comment (anjali-dhanuka) [reply vms/volunteer/views.py]: Fixed it. Please review.

**Comment by anjali-dhanuka** [comment]
Yes @tonythomas01. There was some problem with my editor and I've resolved them now. Please review.

File: vms/volunteer/views.py [hunk 2]
```diff
@@ -25,7 +25,8 @@
 from shift.services import get_volunteer_report, calculate_total_report_hours
 from volunteer.forms import ReportForm, SearchVolunteerForm, VolunteerForm
 from volunteer.models import Volunteer
-from volunteer.services import delete_volunteer_resume, search_volunteers, get_volunteer_resume_file_url
+from volunteer.services import delete_volunteer_resume, search_volunteers,
```
[the remark this answers is not in the document]
Comment (anjali-dhanuka) [reply vms/volunteer/views.py]: This one is done @codacy-bot.

File: vms/volunteer/views.py [hunk 3]
```diff
@@ -25,7 +25,8 @@
 from shift.services import get_volunteer_report, calculate_total_report_hours
 from volunteer.forms import ReportForm, SearchVolunteerForm, VolunteerForm
 from volunteer.models import Volunteer
-from volunteer.services import delete_volunteer_resume, search_volunteers, get_volunteer_resume_file_url
+from volunteer.services import (delete_volunteer_resume, search_volunteers,
+get_volunteer_resume_file_url, has_resume_file, get_volunteer_by_id)
```
[the remark this answers is not in the document]
Comment (anjali-dhanuka) [reply vms/volunteer/views.py]: Yes indentation was required. Done with it. Please review.

File: vms/volunteer/views.py [hunk 4]
```diff
@@ -25,7 +25,9 @@
 from shift.services import get_volunteer_report, calculate_total_report_hours
 from volunteer.forms import ReportForm, SearchVolunteerForm, VolunteerForm
 from volunteer.models import Volunteer
-from volunteer.services import delete_volunteer_resume, search_volunteers, get_volunteer_resume_file_url
+from volunteer.services import (delete_volunteer_resume, search_volunteers,
+                               get_volunteer_resume_file_url, has_resume_file,
```
Comment (tonythomas01) [inline vms/volunteer/views.py]: oopsie. Almost there. 
```
from volunteer.services import (delete_volunteer_resume, search_volunteers,
                                get_volunteer_resume_file_url, has_resume_file,
                                get_volunteer_by_id)

``` 
one space missing on line 29 and 30

Comment (anjali-dhanuka) [reply vms/volunteer/views.py]: @tonythomas01 Done with this too. Please review.
